src/queries/useChunkedCQLFetch.js

```diff
@@ -33,10 +33,12 @@ const useChunkedCQLFetch = ({
   CONCURRENT_REQUESTS = CONCURRENT_REQUESTS_DEFAULT, // Number of requests to make concurrently
   endpoint, // endpoint to hit to fetch items
   generateQueryKey, // Passed function to allow customised query keys
-  ids: passedIds, // List of ids to fetch
+  ids: passedIds, // List of IDs to fetch
+  idName = 'id', // Named ID field to use in the CQL query (i.e. id or userId)
+  limit = 1000, // Item limit to fetch on each request
```

Comment:
It's odd to me that you would ever need this given you are running a query like `id==(foo or bar or bat)`, i.e. retrieving items by an identifier in STEP_SIZE chunks. That is, the reason we're chunking queries with a small step-size is to prevent the query string from growing too-long, and 1000 is already waaaay longer than 60. So I'm just curious about the use-case here. Can you explain?

Reply:
Now that I've grokked folio-org/ui-authorization-roles/pull/32, I understand. If there's a table FOO with PK fooId and a table BAR with PK barId and a join table with multipart PK (fooId, barId), and you're using this tool to query the join table, you would expect the result to contain many more rows than the input query list. 

Sorry that took me so long. I am a bear of very little brain and big queries bother me.

Reply:
@ryandberger, now that I understand it, I see how it's valuable! Up to you whether to leave it in or handle it as a separate feature.

Reply:
I'll leave it in, thanks!